docs/cimpl6ToolingReference.md

```diff
@@ -197,7 +196,7 @@ The Configuration file is a [JSON file](https://www.json.org/) with the followin
 |`fhirTarget`         |<code><i>string</i></code>|The FHIR version this IG will be based on, currently a choice of `"FHIR_R4"`, `"FHIR_STU_3"`, or `"FHIR_DSTU_2"`|
 |`entryTypeURL`       |<code><i>string</i></code>|The root URL for the JSON schema `EntryType` field. **(TO DO: clarify where and how this is used)**   |
 |`filterStrategy`     |`{}` |A JSON object containing configuration for filtering ([see below](#filter-strategy-configuration-parameters)).              |
-|`contentProfile`     |<code><i>string</i></code>| The file name of the content profile for the project.    |
+|`contentProfile`     |<code>ig-<i>**guide-name-FHIR Version**</i>-cp.txt</code>| The file name of the content profile for the project.    |
```

Comment:
The argument is the file name of the content profile -- regardless of whether the recommended pattern for the file name is followed. The user can name the file anything they want. If the pattern isn't followed, the guidance you've added is actually wrong. It was correct as previously written.

Reply:
The update was to align with the naming guidance found in section "Suggested File Naming". Does your comment mean that the information in that section is incorrect?

Reply:
No, the guidance is correct. But the user has to insert the file name of the content profile here -- whether the guidance was followed or not. The string is the type, and the string is the file name of the context profile. I backed out your change.